### PR TITLE
Added solution to Hw hb09

### PR DESCRIPTION
**Pull request by MichaelShevchenko**

### CODE REVIEW

**Review by yutsevich — APPROVED** [review]
Great!

File: src/main/java/Application.java [hunk 1]
```diff
@@ -16,22 +16,23 @@
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.List;
+import org.apache.log4j.Logger;
 
 public class Application {
     private static Injector injector = Injector.getInstance("com.dev.cinema");
+    private static final Logger LOGGER = Logger.getLogger(Application.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/Application.java]:
```suggestion
    private static final Logger logger = Logger.getLogger(Application.class);
```
here and everywhere

File: src/main/java/Application.java [hunk 2]
```diff
@@ -16,22 +16,23 @@
...
         movie1.setTitle("Fast & Furious");
         movie1.setDescription("action");
         movieService.add(movie1);
-        movieService.getAll().forEach(System.out::println);
         Movie movie2 = new Movie();
         movie2.setTitle("Matrix");
         movie2.setDescription("Science fiction");
         movieService.add(movie2);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(movie -> LOGGER.info(movie.toString()));
```
Comment (dmytro-hryhoruk) [inline src/main/java/Application.java]: method reference(toString() will be called automatically

File: src/main/java/com/dev/cinema/dao/CinemaHallDaoImpl.java [hunk 3]
```diff
@@ -5,12 +5,15 @@
...
 import com.dev.cinema.util.HibernateUtil;
 import java.util.List;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger LOGGER = Logger.getLogger(CinemaHallDaoImpl.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/dao/CinemaHallDaoImpl.java]:
```suggestion
    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
```

**Review by dmytro-hryhoruk — COMMENTED** [review]
apply comments to all similar places

File: src/main/java/Application.java (hunk 2 above)
Comment (MichaelShevchenko) [reply src/main/java/Application.java]: done

File: src/main/java/com/dev/cinema/dao/CinemaHallDaoImpl.java (hunk 3 above)
Comment (MichaelShevchenko) [reply src/main/java/com/dev/cinema/dao/CinemaHallDaoImpl.java]: done

File: src/main/java/Application.java (hunk 1 above)
Comment (MichaelShevchenko) [reply src/main/java/Application.java]: done

File: src/main/java/Application.java [hunk 4]
```diff
@@ -16,22 +16,23 @@
...
         movie1.setTitle("Fast & Furious");
         movie1.setDescription("action");
         movieService.add(movie1);
-        movieService.getAll().forEach(System.out::println);
         Movie movie2 = new Movie();
         movie2.setTitle("Matrix");
         movie2.setDescription("Science fiction");
         movieService.add(movie2);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(movie -> logger.info(movie));
```
Comment (dmytro-hryhoruk) [inline src/main/java/Application.java]: method reference

Comment (MichaelShevchenko) [reply src/main/java/Application.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**